src/FixedDataTableContainer.js

```diff
@@ -73,19 +64,19 @@ class FixedDataTableContainer extends React.Component {
 
   render() {
     const fdt = (
-        <FixedDataTable
-            {...this.props}
-            {...this.state}
-            scrollActions={this.scrollActions}
-            columnActions={this.columnActions}
-        />
+      <FixedDataTable
```

Comment:
note for future reviews: it's better to leave out changes like formatting which is not related to the actual ticket.